cookbook/build-admin-frontend.rst

```diff
@@ -77,5 +77,36 @@ on your computer.
     npm install
     npm run build
 
+Common errors
+-------------
+
+If the installation of the npm dependencies or the webpack build fails, you might want to try the following things:
+
+1. Check you Node.js and npm version
+
+You can check the officially supported and tested Node.js and npm version by looking at the `Test Application workflow`_ of the ``sulu/sulu`` package. At the time of writing, this includes Node.js 12, Node.js 14 and npm 6.
+
+.. warning::
+
+    Because of a breaking change for linked packages, Sulu is not compatible with npm v7 at the moment. Have a look at the `issue in the sulu/skeleton repository`_ for more information about this..
+
+2. Clear the npm cache on your machine
+
+The webpack build might fail because of leftovers from previous builds our outdated packages.
+To prevent this, you should remove all the `package-lock.json` files and `node_modules` directories below your project root before installing the npm dependencies:
```

Comment:
```suggestion
To prevent this, you should remove all the ``package-lock.json`` files and ``node_modules`` directories below your project root before installing the npm dependencies:
```